ms-common-impl/src/main/java/net/trajano/ms/vertx/jaxrs/JwtAssertionInterceptor.java

```diff
@@ -136,20 +136,16 @@ public void filter(final ContainerRequestContext requestContext) throws IOExcept
             | ExecutionException e) {
             throw new BadRequestException("unable to parse JWT");
         }
-        if (audience != null) {
+        if (audience != null && !claims.getAudience().contains(audience.toASCIIString())) {
+            LOG.warn("Audience {} did not match {} for {}", claims.getAudience(), audience, requestContext.getUriInfo());
+            requestContext.abortWith(OAuthTokenResponse.unauthorized("invalid_audience", "Audience validation failed", "Bearer").getResponse());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Bearer" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)